src/components/study-pane.js

```diff
@@ -777,6 +777,13 @@ const StudyPane = (props) => {
         }
     }
 
+    function scrollAndOpenVoltageLevel(vlId) {
+        if (!network) return;
+        setDisplayedVoltageLevelId(vlId);
+        showVoltageLevelDiagram(vlId)
+        dispatch(selectItemNetwork(vlId));
```

Comment:
please investigate if we can remove selectItemNetwork and just base highlighting on displayedVoltageLevelId and displayedSubstationId

Reply:
maybe in another PR though

Reply:
setDisplayedVoltageLevelId(vlId)  :  sld displayed
showVoltageLevelDiagram(vlId) : url
dispatch(selectItemNetwork(vlId)) : highlight in network-explorer